_layouts/default.html

```diff
@@ -30,5 +30,14 @@ <h1 class="page-title">Specifying JavaScript.</h1>
           </span>
       </section>
     </main>
+    <div class="popup">
+      <input type="checkbox" id="one" class="hidden" name="ossm">
+      <label for="one" class="alert-message">
+        <strong> <i class="fa fa-heart"></i> Attention</strong> This site is in BETA! We are
+        looking for feed back. Please add issues to our <a
+                    href="https://github.com/tc39/tc39-web-draft/issues" target="_blank">github page</a>!!
+        <span class="close">×</button>
```

Comment:
`<span>...</button>` doesn't seem right to me.
Please add a bit of padding to the button to make it clickable on touch devices.
The touch area is currently only 9px wide and 30px high.